cryptofeed/exchanges/bybit.py

```diff
@@ -481,7 +481,7 @@ async def _order(self, msg: dict, timestamp: float):
                 Decimal(data['price']),
                 Decimal(data['cum_exec_qty']),
                 Decimal(data['qty']) - Decimal(data['cum_exec_qty']),
-                self.timestamp_normalize(data["update_time"]),
+                self.timestamp_normalize(data.get('update_time') or data.get('O') or data.get('timestamp')),
```

Comment:
looks good now. What is the "O" field? I dont think I saw that in the docs?

Reply:
For the spot endpoint, they use O for timestamp I believe
https://bybit-exchange.github.io/docs/spot/#t-executionreport